SVG/stroke-dasharray/xml/KewG_p1-12.xml

```diff
@@ -0,0 +1,220 @@
+
```

Comment:
@Yuying-Jin Suggestion: Let's condense the XML into just one file, and simplify it so it's easy for someone to read if they are just interested in the tutorial.